src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        String color = new ColorSupplier().getRandomColor();
+        int number = new Random().nextInt(MAX_NUMBER);
```

Comment:
number is not very good variable name, occures question what number? what is the purpose of it? etc